doc/nrf/releases_and_maturity/known_issues.rst

```diff
@@ -2217,6 +2217,10 @@ CIA-351: Connectivity issues with :ref:`lib_azure_iot_hub`
 Other libraries
 ===============
 
+.. rst-class:: v2-4-1 v2-4-0 v2-3-0 v2-2-0 v2-1-4 v2-1-3 v2-1-2 v2-1-1 v2-1-0 v2-0-2 v2-0-1 v2-0-0 v1-9-2 v1-9-1 v1-9-0 v1-8-0 v1-7-1 v1-7-0 v1-6-1 v1-6-0 v1-5-2 v1-5-1 v1-5-0 v1-4-2 v1-4-1 v1-4-0 v1-3-2 v1-3-1 v1-3-0 v1-2-1 v1-2-0 v1-1-1 v1-0-0 v0-4-0
+
+NCSDK-22908: The :ref:`st25r3911b_nfc_readme` library returns processing error in case when the Rx complete event is received together with the FIFO water level event.
```

Comment:
```suggestion
NCSDK-22908: The :ref:`st25r3911b_nfc_readme` library returns a processing error in case the Rx complete event is received together with the FIFO water level event.
```